client/ayon_core/plugins/publish/collect_anatomy_instance_data.py

```diff
@@ -411,13 +415,18 @@ def _fill_folder_data(self, instance, project_entity, anatomy_data):
             if hierarchy:
                 parent_name = hierarchy.split("/")[-1]
 
-            folder_name = instance.data["folderPath"].split("/")[-1]
+            folder_path = instance.data["folderPath"]
+            folder_name = folder_path.split("/")[-1]
             anatomy_data.update({
                 "asset": folder_name,
                 "hierarchy": hierarchy,
                 "parent": parent_name,
                 "folder": {
                     "name": folder_name,
+                    "path": folder_path,
+                    "type": "Shot",
```

Comment:
Yeah - this doesn't really make sense to have this hardcoded type? Types are dynamic and a Shot type may not even be defined in the current project.

Reply:
Oh, yeah, that's actually comming from current editorial logic, it's hardcoded there. Should be changed, out of scope of this PR.